substrate/src/src/lib.rs

```diff
@@ -1,297 +0,0 @@
-// This file is part of Substrate.
```

Comment:
was deleting this file intentional?

Reply:
As far as i can tell it was a duplicate of `substrate/src/lib.rs`. Dont think we need this 🤔